templates/_macros/rich-vertical-list.jinja

```diff
@@ -0,0 +1,127 @@
+# Params
+# title (string) (required): Title of the rich vertical list
+# flipped (boolean) (optional): Whether the list items are flipped so image is on the left and the text is on the right. Defaults to false.
```

Comment:
Just for clarification, are you looking for the integer variant approach as explored in #5304 ? If so, I think we should have a wider team discussion on this approach before committing to it

Reply:
I don't want to be blocking this on the decision. We already have couple patterns merged that use flags for variants, so this follows existing convention.

Obviously, it would be good to have a conclusion before we officially publish macros (not to change API later), but I think we are fine doing any changes if needed before that.